tensorflow/compiler/xla/service/gpu/ir_emission_utils.h

```diff
@@ -157,6 +157,11 @@ bool ImplementedAsLibraryCall(const HloInstruction& hlo);
 // kept are contiguous in the input of the reduce instruction.
 bool IsReductionFromOrToContiguousDimensions(const HloInstruction& reduce);
 
+// Whether it is an input fusion whose root is either a non-strided slice
+// or a tuple of non-strided slices.
+bool IsInputFusibleSlices(const HloInstruction& unnested_hlo,
```

Comment:
Could you document the semantics of `verify_no_strides`?

Reply:
No problem.

Reply:
"verify" is not clear, how about something like

`Returns whether input is either an input fusion [...]. If verify_no_strides is true, returns false unless ...`

Reply:
Good suggestion. Taken.

Reply:
Sorry for nitpicking, now the first two sentences almost duplicate each other. Can we write "Returns ..." with all the relevant information available after?

Reply:
My oversight. Will do.

Reply:
Done.